src/main/java/net/simpvp/PvPTeleport/PvPTransportation.java

```diff
@@ -64,14 +59,15 @@ private static Location randomSpawn() {
 
 		Random RNG = new Random();
 
-		Material blocktype = null;
+		Material blocktype;
```

Comment:
If you're cleaning this up you could move the variable declaration down to where it gets assigned a value.